.github/workflows/bundle-release.js

```diff
@@ -44,9 +44,10 @@ const packageJson = JSON.stringify(
     repository: mainPackageJson.repository,
     main: "js/index.js",
     scripts: {
-      postinstall: rewritePrefix
-        ? "ESY_RELEASE_REWRITE_PREFIX=true node ./postinstall.js"
-        : "node ./postinstall.js"
+      postinstall:
+        rewritePrefix ?
+	"node -e \"process.env['OCAML_VERSION'] = '4.9.1000'; process.env['OCAML_PKG_NAME'] = 'ocaml'; process.env['ESY_RELEASE_REWRITE_PREFIX']=true; require('./postinstall.js')\"":
```

Comment:
Later, if you wish the ppx to be statically built (assuming natdynlink is not an issue here), the environment variables can be conditionally set like how esy does

```sh
"node -e \"process.env['OCAML_VERSION'] = process.platform == 'linux' ? '4.10.1002-musl.static.flambda': '4.10.0';\"'
```

https://github.com/esy/esy/blob/master/.ci/pipelines-release.js#L52